posteriors/laplace/dense_hessian.py

```diff
@@ -0,0 +1,144 @@
+from typing import Any, NamedTuple
+from functools import partial
+import torch
+from optree import tree_map
+from optree.integration.torch import tree_ravel
+
+from posteriors.types import TensorTree, Transform, LogProbFn
+from posteriors.tree_utils import tree_size
+from posteriors.utils import (
+    is_scalar,
+    CatchAuxError,
+)
+
+from torch.func import jacrev, jacfwd
+
+
+def build(
+    log_posterior: LogProbFn,
+    init_prec: torch.Tensor | float = 0.0,
```

Comment:
`epsilon` and `rescale` for the `update` function should be included here too

Reply:
Added!